### PR TITLE
Add org.freedesktop.Sdk.Extension.swift6

### PR DESCRIPTION
**Pull request by david-swift**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
  
The SDK Extension for the Swift programming language version 6. This is the successor of the [SDK Extension for Swift 5.](https://flathub.org/apps/org.freedesktop.Sdk.Extension.swift5)

- [ ] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [ ] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by david-swift** [comment]
bot, build org.freedesktop.Sdk.Extension.swift6

File: org.freedesktop.Sdk.Extension.swift6.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		    "type": "archive",
+		    "url": "https://github.com/realm/SwiftLint/releases/download/0.57.0/swiftlint_linux.zip",
+		    "sha256": "def1803e705c738c97fbc275c34e7a2e0278944776a8b9c56ebe8c444ea57407"
+		}
+	    ],
+	    "build-commands": [
+		"cp swiftlint /usr/lib/sdk/swift6/bin/"
+	    ]
+	},
+        {
```
Comment (bbhtt) [inline org.freedesktop.Sdk.Extension.swift6.json:62]: can you fix the indent here ?

File: org.freedesktop.Sdk.Extension.swift6.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                {
+                    "type": "git",
+                    "url": "https://github.com/swiftlang/swift-format.git",
+                    "commit": "65f9da9aad84adb7e2028eb32ca95164aa590e3b"
+                },
+                "generated-sources.json"
+            ],
+            "build-commands": [
+            	"./setup-offline.sh",
+                "/usr/lib/sdk/swift6/bin/swift build -c release --skip-update",
```
Comment (bbhtt) [inline org.freedesktop.Sdk.Extension.swift6.json:75]: indent here too

**Comment by david-swift** [comment]
bot, build org.freedesktop.Sdk.Extension.swift6

**Comment by david-swift** [comment]
Thank you!

**Pull request closed without merging**